src/engine/ruleset/decoders/f5/f5-apm.yml

```diff
@@ -0,0 +1,408 @@
+---
+name: decoder/f5-apm/0
+
+metadata:
+  module: F5
+  title: F5 APM Decoder
+  description: Decodes F5 BIG-IP Access Policy Manager logs
+  compatibility: >
+    This module has been tested against F5 BIG-IP version 16.1.0
+  author:
+    name: Wazuh, Inc.
+    date: 2023/02/04
+  references:
+    - https://techdocs.f5.com/kb/en-us/products/big-ip_apm/manuals/product/apm-network-access-12-1-0/10.html
+
+sources:
+  - decoder/queue-syslog/0
+  - decoder/queue-localfile/0
+
+definitions:
+  def_session : :<~>:<~/ignore/ ><~tmp.rsa.misc.log_session_id>
```

Comment:
In the other parsers where we use definitions, the names were written in uppercase, It isn't mandatory and it wasn't even defined as good practice, but I guess that it could help users while reading.